src/Auxiliary/Aux_Check_Parameter.cpp

```
@@ -87,8 +87,8 @@ void Aux_Check_Parameter()
    if ( END_STEP < 0  &&  OPT__INIT != INIT_BY_RESTART )
       Aux_Error( ERROR_INFO, "incorrect parameter \"%s = %d\" [>=0] !!\n", "END_STEP", END_STEP );
 
-   if ( END_T < 0.0  &&  OPT__INIT != INIT_BY_RESTART )
-      Aux_Error( ERROR_INFO, "incorrect parameter \"%s = %14.7e\" [>=0] !!\n", "END_T", END_T );
+   if ( END_T < INIT_TIME  &&  OPT__INIT != INIT_BY_RESTART )
```

Clang (14.0.6)

[note] +1, including nesting penalty of 0, nesting level increased to 1

(clang)